content/docs/reference/autocert.mdx

```diff
@@ -15,47 +17,42 @@ import TabItem from '@theme/TabItem';
 
-This reference covers all of Pomerium's **Autocert Settings**:
+Pomerium’s Autocert settings provide a certificate issuance and management service in accordance with the [automated certificate management environment](https://letsencrypt.org/2019/03/11/acme-protocol-ietf-standard.html) (ACME) protocol.
 
-- [Autocert](#autocert)
-- [Autocert CA](#autocert-ca)
-- [Autocert Directory](#autocert-directory)
-- [Autocert EAB Key ID](#autocert-eab-key-id)
-- [Autocert EAB Mac Key](#autocert-eab-mac-key)
-- [Autocert Email](#autocert-email)
-- [Autocert Must Staple](#autocert-must-staple)
-- [Autocert Trusted Certificate Authority](#autocert-trusted-certificate-authority)
-- [Autocert Use Staging](#autocert-use-staging)
+Autocert can be useful in situations where Pomerium may not be behind a TLS-terminating ingress controller, proxy, or load balancer that manages TLS certificates on behalf of your upstream services. Certificates provisioned by Autocert provide also [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling).
 
-## Autocert {#autocert}
+:::note Autocert support
+
+The suite of Autocert settings listed below are supported only in [Pomerium Core](/docs/core) configurations. Here's what you should know about ACME support in other Pomerium products:
 
-Turning on Autocert allows Pomerium to automatically retrieve, manage, and renew public facing TLS Certificates from [Let's Encrypt][letsencrypt], which includes managed routes and the Authenticate Service.
+- Pomerium Enterprise users should note that Autocert cannot be configured in the [Enterprise Console](/docs/enterprise#enterprise-console) or the Pomerium Enterprise [configuration file](/docs/enterprise/configure). Autocert is a bootstrap setting, meaning it must be configured in Pomerium Core (either in a configuration file or as an environment variable) before running Pomerium Enterprise.
 
-You must use [Autocert Directory](/docs/reference/autocert) with Autocert to store and share certificate data between services. (Note that Autocert also provides [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling).)
+- Kubernetes users should refer to the [Cert-Manager integration](/docs/k8s/ingress#cert-manager-integration) section for more information on configuring automated certificate management in the Pomerium Ingress Controller.
 
-The Autocert setting can be useful in situations where you may not have Pomerium behind a TLS terminating ingress or proxy that is already handling your public certificates on your behalf.
+- Pomerium Zero users should note that Pomerium automatically manages certificates for [starter domains](/docs/concepts/clusters#starter-domain) and [custom domains](/docs/capabilities/custom-domains).
 
-Autocert will incorporate certificates available in the system trust store and certificates set manually in the Pomerium configuration, and they will take precedence over generated certificates when applicable to configured routes.
+:::
+
+## Autocert {#autocert}
```

Comment:
I think a lot of the information in this section could be moved up into the overall summary.

High level comment: I think we should aim to structure this so we put the most important information first, and I'm not sure we achieve that so much of the information is tucked away under the setting headers and the headers are ordered alphabetically.

Reply:
I feel like there are two approaches to this:
- Include this information at the top and less details in the settings themselves. This is ideal assuming most users start at the top and work their way down.
- Include the pertinent details under the settings themselves, not the top, in case users enter a page by Algolia search or a reference page filtered search. In this case, they'd miss the details at the top.

I'd imagine most users would land on the page and scroll to the setting without reading the summary, or be directed to a setting from the reference page or Algolia search  to read how to configure it. 

In the case of Autocert and Autocert Directory, they'd then see that Autocert must be used in conjunction with the Autocert Directory setting. If you provide this at the top of the page, users might skim over it or miss it entirely. 

I'm not opposed to moving it to the top, but this is something I considered when making changes. What are your thoughts here?

Reply:
The more I think about it, though, logically you'd expect to see the most critical details at the top of a page, not buried under a setting you may miss. Happy to discuss more on Slack or a quick call, though.

Reply:
Updated.